src/qfm/pricing/pricing.cpp

```diff
@@ -23,9 +23,9 @@ void Pricing::SetModel(std::shared_ptr<model::Model> model,
 
 double Pricing::GetAssetPrice(
     std::shared_ptr<asset::Asset> asset) const noexcept {
-  auto it = models_.find(asset->GetType());
-  if (it != models_.end()) {
-    return it->second->GetAssetPrice(asset);
+  if (auto iterator = models_.find(asset->GetType());
+      iterator != models_.end()) {
```

Comment:
I'm not sure I like it because it looks less readable. Can you point to the best practices?

Reply:
Reverted to minimize changes — only thing `clang-tidy` was marking as warning was short variable name, hence I renamed from `it` to `iterator`.

```text
/Volumes/workplace/quant_finance_models/src/qfm/pricing/pricing.cpp:26:8: error: variable name 'it' is too short, expected at least 3 characters [readability-identifier-length,-warnings-as-errors]
   26 |   auto it = models_.find(asset->GetType());
      |        ^
```

Reply:
It makes sense, thanks.